evals/registry/evals/speech_translation_tasks.yaml

```diff
@@ -0,0 +1,1084 @@
+### Audio translation tasks (X to EN)
```

Comment:
maybe call this file `audio_translation_tasks.yaml` so it's consistent with `audio_tasks.yaml`